src/main/java/com/microsoft/azure/vmagent/AzureVMAgent.java

```diff
@@ -181,7 +183,13 @@ public AzureVMAgent(
             String remotingOptions,
             AzureVMAgentTemplate template) throws FormException, IOException {
 
-        super(name, nodeDescription, remoteFS, numExecutors, mode, label, launcher, retentionStrategy, nodeProperties);
+        super(name, remoteFS, launcher);
```

Comment:
migrated to non deprecated API, required so that I could customise the node properties